### PR TITLE
Add support for CSD3

### PR DESCRIPTION
**Pull request by samueljackson92**
This removes the segfault handling code and adds jobscripts for CSD3.

To review:
 - Follow the setup & running instructions in the README
 - Code review.

Note use: 

```sh
python -m pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple uda
```

to upgrade uda

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +46,23 @@ source ~/rds/rds-ukaea-mast-sPGbyCAPsJI/uda-ssl.sh
...
 You should now be able to run the following commands.
 
-## Local Ingestion
+### Submitting runs on CSD3
+
+1. First submit a job to collect all the metadata:
+
+```sh
+sbatch ./jobs/metadata.csd3.slurm.sh
+```
```
Comment (jameshod5) [inline README.md:55]: if you want it more foolproof perhaps add something to `cd ..` out of the mastcodes folder, the above step puts the user in mastcodes and not in the directory this step requires

File: README.md [hunk 2]
```diff
@@ -45,7 +46,23 @@ source ~/rds/rds-ukaea-mast-sPGbyCAPsJI/uda-ssl.sh
...
+
+2. Then submit an ingestion job
+
+```sh
+sbatch ./jobs/ingest.csd3.slurm.sh campaign_shots/tiny_campaign.csv s3://mast/test/shots/ amc
```
Comment (jameshod5) [inline README.md:60]: User will need the .s5cfg file, not sure if that should be specified in the readme?

**Review by jameshod5 — CHANGES_REQUESTED** [review]
`python -m pip install --U pip` - This doesn't work for me, have to use --upgrade instead. (or change to -U instead)

other two comments are more suggestions, but the above needs to be fixed

**Comment by samueljackson92** [comment]
I had added your suggestions to the README.

**Comment by jameshod5** [comment]
> `python -m pip install --U pip` - This doesn't work for me, have to use --upgrade instead. (or change to -U instead)
> 
> other two comments are more suggestions, but the above needs to be fixed

This still needs to change to `python -m pip install --upgrade pip` or `python -m pip install -U pip`

**Comment by samueljackson92** [comment]
Sorry missed that one!

**Comment by jameshod5** [comment]
Looks good, merged

**Comment by samueljackson92** [comment]
[heart]         Jackson, Samuel (Computing) reacted to your message:
________________________________

**Pull request merged**